src/main/java/pl/allegro/tech/embeddedelasticsearch/EmbeddedElastic.java

```diff
@@ -194,29 +194,38 @@ public void refreshIndices() {
 
     public static final class Builder {
 
-        private int portNumber = 9300;
-        private String clusterName = "elasticsearch";
+        private Map<String, Object> settings = new HashMap<>();
         private Optional<String> version = Optional.empty();
         private List<InstallationDescription.Plugin> plugins = new ArrayList<>();
         private Optional<URL> downloadUrl = Optional.empty();
         private Map<String, IndexSettings> indices = new HashMap<>();
 
         private Builder() {
+            this.settings.put("cluster.name", "elasticsearch");
```

Comment:
Ok, but I suggest not to use guava. In this project guava comes from elasticsearch dependency but future versions do not use this library.